app/Http/Controllers/AtomController.php

```diff
@@ -60,9 +60,19 @@ private function feedAddItem(&$feed, $incident)
         $feed->add(
             $incident->name,
             Setting::get('app_name'),
-            Setting::get('app_domain'),
+            $this->canonicalizeUrl(Setting::get('app_domain')).'#'.$incident->id,
             $incident->created_at->toAtomString(),
             $incident->message
         );
     }
+
+    /**
+     * Add a / at the end of an URL to in order to be W3C compliant
+     *
+     * @param string $url
+     */
+    private function canonicalizeUrl($url)
+    {
+        return preg_replace('/([^\/])$/', '$1/', $url);
```

Comment:
Where should I put it? Creating a directory Traits in `app/Http`? I'm not a PHP guy, I don't know the conventions.